framework/auth/core.py

```diff
@@ -56,7 +56,6 @@ def string_required(value):
 
 
 def validate_history_item(item):
-    string_required(item.get('institution'))
```

Comment:
When you delete the entire object, then the institution will be cleared along with all other fields.